pkg/cvo/updatepayload.go

```diff
@@ -84,24 +84,27 @@ func (r *payloadRetriever) RetrievePayload(ctx context.Context, update configv1.
 	var info PayloadInfo
 
 	// verify the provided payload
-	var releaseDigest string
-	if index := strings.LastIndex(update.Image, "@"); index != -1 {
-		releaseDigest = update.Image[index+1:]
-	}
-	if err := r.verifier.Verify(ctx, releaseDigest); err != nil {
-		vErr := &payload.UpdateError{
-			Reason:  "ImageVerificationFailed",
-			Message: fmt.Sprintf("The update cannot be verified: %v", err),
-			Nested:  err,
+	/*
+		var releaseDigest string
+		if index := strings.LastIndex(update.Image, "@"); index != -1 {
+			releaseDigest = update.Image[index+1:]
 		}
-		if !update.Force {
-			return PayloadInfo{}, vErr
+		if err := r.verifier.Verify(ctx, releaseDigest); err != nil {
+			vErr := &payload.UpdateError{
+				Reason:  "ImageVerificationFailed",
+				Message: fmt.Sprintf("The update cannot be verified: %v", err),
+				Nested:  err,
+			}
+			if !update.Force {
+				return PayloadInfo{}, vErr
+			}
+			klog.Warningf("An image was retrieved from %q that failed verification: %v", update.Image, vErr)
+			info.VerificationError = vErr
+		} else {
+			info.Verified = true
 		}
-		klog.Warningf("An image was retrieved from %q that failed verification: %v", update.Image, vErr)
-		info.VerificationError = vErr
-	} else {
-		info.Verified = true
-	}
+	*/
```

Comment:
This is just commented out for testing right?  We should drop the comments and remove the indent injection now that you have things working locally.